examples/advanced/custom-transforms/build.js

```diff
@@ -96,7 +96,7 @@ StyleDictionary.registerTransformGroup({
 StyleDictionary.registerFormat({
   name: 'custom/android/xml',
   formatter: function(dictionary) {
-    return dictionary.allTokens.map(function(token) {
+    return dictionary.allProperties.map(function(token) {
```

Comment:
This one is my bad. We are moving to using "design tokens" and "tokens" rather than "properties" in 3.0. This change will be backwards compatible (you can still use `allProperties` here), but we wanted to update the examples to reflect this change. The reason this doesn't work is I didn't push a pre-release version after making that change. If you rebase (I pushed a pre-release version) and change this back to `allTokens` and update line 98 to use a de-structured argument: `formatter: function({ dictionary }) {`

Reply:
ah ok - will revert this 👍